app/services/container_dashboard_service.rb

```diff
@@ -99,9 +99,14 @@ def providers
   end
 
   def alerts
-    provider_ids = ManageIQ::Providers::ContainerManager.all.pluck(:id)
-    relation = @ems ? @ems.miq_alert_statuses : MiqAlertStatus.where(:ems_id => provider_ids)
-    alerts_status = relation.present? ? relation.where(:resolved => [false, nil]).group(:severity).count.values_at('error', 'warning') : [nil, nil]
+    providers = @ems.present? ? [@ems] : ManageIQ::Providers::ContainerManager.includes(:authentications)
+
+    has_alerts = providers.any? do |prov|
+      prov.authentications.any? { |x| x.authtype == 'prometheus_alerts' }
```

Comment:
`prov.monitoring_manager != nil`

Reply:
(the existence of the monitoring manager should encapsulate the authentication/endpoint)